README.md

```diff
@@ -1,81 +1,85 @@
+Stay tuned!
 
-## Installation
+## ⚙️ Installation
 
-For now, the only supported method to use skore is from source.
-Follow the instructions in [CONTRIBUTING.md](/CONTRIBUTING.md#quick-start) to install dependencies and start the UI.
-
-## Quick start
+You can install `skore` by using `pip`:
+```bash
+pip install -U skore
+```
 
-For a complete introductory example, see our [basic usage notebook](/examples/basic_usage.ipynb). The resulting skore report has been exported to [this HTML file](https://gist.github.com/augustebaum/6b21dbd7f7d5a584fbf2c1956692574e): download it and open it in your browser to visualize it.
+## 🚀 Quick start
 
 In your shell, run the following to create a project file `project.skore` (the default) in your current working directory:
-```sh
+```bash
 python -m skore create
 ```
 
-Run the following in your Python code to load the project:
+Run the following in your Python code (in the same working directory) to load the project, store some objects, delete them, etc:
 ```python
 from skore import load
 
+# load the project
 project = load("project.skore")
-```
 
-You can save items you need to track in your project:
-```python
+# save an item you need to track in your project
 project.put("my int", 3)
-```
 
-You can also get them back:
-```python
+# get an item's value
 project.get("my int")
-```
 
-By default, strings are assumed to be Markdown:
-```python
+# by default, strings are assumed to be Markdown:
 project.put("my string", "Hello world!")
-```
 
-Note that `put` overwrites previous data
-```python
+# `put` overwrites previous data
 project.put("my string", "Hello again!")
-```
 
-You can list all the keys in a project with:
-```python
-project.list_keys()
-```
+# list all the keys in a project
+print(project.list_keys())
 
-You can delete items with:
-```python
+# delete an item
 project.delete_item("my int")
 ```
 
-Then, in the directory containing your project, run the following command to start the UI locally:
-```sh
+Then, in the directory containing your project, run the following command in your shell to start the UI locally:
+```bash
 python -m skore launch project.skore
 ```
-
 This will automatically open a browser at the UI's location.
+You can then drag and drop items into the dashboard and start editing your report.
+
+💡 Note that after launching the dashboard, you can keep modifying current items or store new ones, and the dashboard will automatically be refreshed.
 
+👨‍🏫 For a complete introductory example, see our [basic usage notebook](/examples/basic_usage.ipynb).
+It shows you how to store all types of items: python lists and dictionaries, `numpy` arrays, `scikit-learn` fitted models, `matplotlib` graphs, etc.
+The resulting `skore` report has been exported to [this HTML file](https://gist.github.com/augustebaum/6b21dbd7f7d5a584fbf2c1956692574e): download it and open it in your favorite browser to visualize it.
```

Comment:
Could we host this output in a place where it will render properly. Sending user to a compressed HTML output seems counter productive to me.

We may use github pages ?

Reply:
I did some tests in a separate repo, but I have some issues with HTML exported from `skore`:

https://github.com/sylvaincom/html_preview

Reply:
I was trying an alternative to GitHub Pages called "html-preview", but ok I'll do GitHub Pages then